code/__DEFINES/movespeed_modification.dm

```diff
@@ -1,69 +1,71 @@
-#define MOVESPEED_ID_DIE_OF_FATE                        "DIE_OF_FATE"
-
-#define MOVESPEED_ID_SHOVE                              "SHOVE"
-#define MOVESPEED_ID_BULKY_DRAGGING                     "BULKY_DRAG"
-#define MOVESPEED_ID_HUMAN_CARRYING                     "HUMAN_CARRY"
-#define MOVESPEED_ID_SHRINK_RAY							"SHRUNKEN_SPEED_MODIFIER"
+#define MOVESPEED_DATA_INDEX_PRIORITY 1
+#define MOVESPEED_DATA_INDEX_FLAGS 2
+#define MOVESPEED_DATA_INDEX_MULTIPLICATIVE_SLOWDOWN 3
+#define MOVESPEED_DATA_INDEX_MOVETYPE 4
+#define MOVESPEED_DATA_INDEX_BL_MOVETYPE 5
+#define MOVESPEED_DATA_INDEX_CONFLICT 6
+
+#define MOVESPEED_DATA_INDEX_MAX 6
+
+//flags
+#define IGNORE_NOSLOW	(1 << 0)
+
+//conflict types
+
+#define MOVE_CONFLICT_JETPACK							"JETPACK"
+
+//ids
+
+#define MOVESPEED_ID_MOB_WALK_RUN_CONFIG_SPEED			"MOB_WALK_RUN"
+#define MOVESPEED_ID_CONFIG_SPEEDMOD					"MOB_CONFIG_MODIFIER"
+
+#define MOVESPEED_ID_SLIME_REAGENTMOD					"SLIME_REAGENT_MODIFIER"
+#define MOVESPEED_ID_SLIME_HEALTHMOD					"SLIME_HEALTH_MODIFIER"
+#define MOVESPEED_ID_SLIME_TEMPMOD						"SLIME_TEMPERATURE_MODIFIER"
+
+#define MOVESPEED_ID_SLIME_STATUS						"SLIME_STATUS"
+
+#define MOVESPEED_ID_TARANTULA_WEB						"TARANTULA_WEB"
+
+#define MOVESPEED_ID_LIVING_TURF_SPEEDMOD				"LIVING_TURF_SPEEDMOD"
+
+#define MOVESPEED_ID_CARBON_SOFTCRIT					"CARBON_SOFTCRIT"
+#define MOVESPEED_ID_CARBON_OLDSPEED					"CARBON_DEPRECATED_SPEED"
+#define MOVESPEED_ID_CARBON_CRAWLING					"CARBON_CRAWLING"
+
+#define MOVESPEED_ID_DNA_VAULT							"DNA_VAULT"
+
+#define MOVESPEED_ID_YELLOW_ORB							"YELLOW_ORB"
+
+#define MOVESPEED_ID_TARFOOT							"TARFOOT"
+
+#define MOVESPEED_ID_SEPIA								"SEPIA"
+
+#define MOVESPEED_ID_MONKEY_REAGENT_SPEEDMOD			"MONKEY_REAGENT_SPEEDMOD"
+#define MOVESPEED_ID_MONKEY_TEMPERATURE_SPEEDMOD		"MONKEY_TEMPERATURE_SPEEDMOD"
+#define MOVESPEED_ID_MONKEY_HEALTH_SPEEDMOD				"MONKEY_HEALTH_SPEEDMOD"
+
+#define MOVESPEED_ID_CHANGELING_MUSCLES					"CHANGELING_MUSCLES"
+
+#define MOVESPEED_ID_SIMPLEMOB_VARSPEED					"SIMPLEMOB_VARSPEED_MODIFIER"
+#define MOVESPEED_ID_ADMIN_VAREDIT						"ADMIN_VAREDIT_MODIFIER"
+
+#define MOVESPEED_ID_PAI_SPACEWALK_SPEEDMOD				"PAI_SPACEWALK_MODIFIER"
+
+#define MOVESPEED_ID_SANITY								"MOOD_SANITY"
+
+#define MOVESPEED_ID_SPECIES							"SPECIES_SPEED_MOD"
+#define MOVESPEED_ID_SNAIL_CRAWL						"SNAIL_CRAWL_SPEED_MOD"
+
+#define MOVESPEED_ID_CYBER_THRUSTER						"CYBER_IMPLANT_THRUSTER"
+#define MOVESPEED_ID_JETPACK							"JETPACK"
+
+#define MOVESPEED_ID_SLAUGHTER                          "SLAUGHTER"
+#define MOVESPEED_ID_DIE_OF_FATE                        "DIE_OF_FATE"
+
+#define MOVESPEED_ID_SHOVE                              "SHOVE"
+#define MOVESPEED_ID_BULKY_DRAGGING                     "BULKY_DRAG"
+#define MOVESPEED_ID_HUMAN_CARRYING                     "HUMAN_CARRY"
+#define MOVESPEED_ID_SHRINK_RAY							"SHRUNKEN_SPEED_MODIFIER"
+#define MOVESPEED_ID_PEPPER_SPRAY                       "PEPPER_SPRAYED"
```

Comment:
wtf

Reply:
I didnt want to do it reeee

Reply:
All I wanted was a quick fix dammit

Reply:
Do `git checkout upstream/master -- code/__DEFINES/movespeed_modification.dm` to undo changes to this file

Reply:
Yeah Ill do git pull upstream master first then try that